src/app/+admin/admin-sidebar/admin-sidebar.component.ts

```diff
@@ -104,32 +113,38 @@ export class AdminSidebarComponent extends MenuComponent implements OnInit {
         active: false,
         visible: true,
         model: {
-          type: MenuItemType.LINK,
+          type: MenuItemType.ONCLICK,
           text: 'menu.section.new_community',
-          link: '/communities/submission'
-        } as LinkMenuItemModel,
+          function: () => {
+            this.modalService.open(CreateCommunityParentSelectorComponent);
+          }
+        } as OnClickMenuItemModel,
```

Comment:
Strangely, even though Create Community is listed *first* in this component, it shows up *last* under the "New" menu.  I'm not sure that's a fault of this PR though.

Reply:
Strangely enough, this behaviour only occurs when preboot is enabled and is not related to this PR.
I'm not sure why this happens, but before launching the [Admin Menu PR](https://github.com/DSpace/dspace-angular/pull/338) I discussed this with @artlowel and we decided it's not really a bug: if you want to force a specific order of menu items, you can explicitly pass a manual "index" when creating a new menu item.

Reply:
I agree this isn't the fault of this PR.  I see the same behavior on `master`.

However, I'm going to log it as a bug, as I feel these menus should retain an expected ordering. At the very least, we need to document the "force a specific order of menu items" option you noted, but ideally we should find a way to get ordering to remain consistent.

Nonetheless, it's a bug not caused by this PR. So, this PR can obviously move forward as-is.